app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<div style="
+min-height: 60px;
+display: flex;
+align-items: center;
+justify-content: space-between;
+padding: 0 20px;
+border-bottom: 1px solid;
+">
+<strong>
+<%= @category.name%>
+</strong>
+<strong>
+<% total_amount = Transaction.where(category_id: @category.id).sum(:amouunt)%>
+<%= number_to_currency(total_amount, :unit => "$")%>
+</strong>
+</div>
+<div style="text-align: center; padding: 8px 0">
+<h1>Transactions</h1>
+</div>
+<div id="transactions">
+  <% @transactions.includes(:category).each do |transaction| %>
```

Comment:
- [ ] Based on the requirements of this project, for a given category, the list of transactions is presented ordered by the most recent. Kindly rectify :wrench: this issue.

:smile:  Since we are tracking our expenses what do you think of adding the date of creation of each transaction?

Reply:
* Resolved 😉  👍🏻  The Date has been added as suggested by the previous reviewer. Congrats @Sevenpros 🎉